features/lib3mf.prf

```diff
@@ -0,0 +1,49 @@
+# Detect lib3mf, then use this priority list to determine
+# which library to use:
+#
+# Priority
+# 1. LIB3MF_INCLUDEPATH / LIB3MF_LIBPATH (qmake parameter, not checked if given on commandline)
+# 2. OPENSCAD_LIBRARIES (environment variable)
+# 3. system's standard include paths
+
+OPENSCAD_LIBRARIES_DIR = $$(OPENSCAD_LIBRARIES)
+
+!isEmpty(LIB3MF_INCLUDEPATH) {
+  LIB3MF_CFLAGS = -I$$LIB3MF_INCLUDEPATH -I$$LIB3MF_INCLUDEPATH/Model/COM
+} else {
+  !isEmpty(OPENSCAD_LIBRARIES_DIR) {
+    exists($$OPENSCAD_LIBRARIES_DIR/include/lib3mf/Model/COM/NMR_DLLInterfaces.h) {
+      LIB3MF_CFLAGS = -I$$OPENSCAD_LIBRARIES_DIR/include/lib3mf -I$$OPENSCAD_LIBRARIES_DIR/include/lib3mf/Model/COM
```

Comment:
It seems a bit odd to add some subpath under 3mf as our include path. Couldn't we just `#include <lib3mf/Model/COM/NMR_DLLInterfaces.h>` in our code?

Reply:
They had lots of internal references that need the multiple `-I` flags. Maybe that is fixed now and just the main `.../lib3mf` is enough.

Reply:
Yep, that's fine now, I changed our code too, so the Model/COM deep link is not needed anymore.